src/decimal.rs

```diff
@@ -273,6 +276,73 @@ impl TryMul<Decimal> for Decimal {
     }
 }
 
+#[cfg(feature = "ops-traits")]
+impl<T> Mul<T> for Decimal
+where
+    T: Into<U192>,
+{
+    type Output = Decimal;
+    fn mul(self, rhs: T) -> Decimal {
+        self.try_mul(rhs).unwrap()
+    }
+}
```

Comment:
Correct me if I'm wrong but doesn't this allows
```
let a: u64 = 3;
let b: Decimal = Decimal::from(4);
let c = b * a;
// c != 12 !!
```

Which is bad IMO?

Reply:
it yields Decimal::from(12), added some unit tests so I exemplify it.